Mathlib/Analysis/InnerProductSpace/Spectrum.lean

```diff
@@ -182,59 +191,84 @@ end Version1
 
 section Version2
 
-variable {n : ℕ} (hn : Module.finrank 𝕜 E = n)
-
-/-- A choice of orthonormal basis of eigenvectors for self-adjoint operator `T` on a
-finite-dimensional inner product space `E`.
-
-TODO Postcompose with a permutation so that these eigenvectors are listed in increasing order of
-eigenvalue. -/
-noncomputable irreducible_def eigenvectorBasis : OrthonormalBasis (Fin n) 𝕜 E :=
-  hT.direct_sum_isInternal.subordinateOrthonormalBasis hn hT.orthogonalFamily_eigenspaces'
-
-/-- The sequence of real eigenvalues associated to the standard orthonormal basis of eigenvectors
-for a self-adjoint operator `T` on `E`.
+variable {n : ℕ}
 
-TODO Postcompose with a permutation so that these eigenvalues are listed in increasing order. -/
-noncomputable irreducible_def eigenvalues (i : Fin n) : ℝ :=
+/--
+Unsorted eigenvalues and eigenvectors.  These are composed with a permutation below. -/
```

Comment:
Can you direct people to use `eigenvalues` instead of this in the docstring (instead of just saying "below")?